Menu/Menu.js

```diff
@@ -1,10 +1,12 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector(".menu");
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelectorAll(".menu-button");
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click', () => toggleMenu());
```

Comment:
Or you could keep using querySelectorAll, and just grab the one and only node in the node list like this. 

```suggestion
menuButton[0].addEventListener('click', () => toggleMenu());
```